tests/unit/sagemaker/model/test_neo.py

```diff
@@ -286,7 +282,7 @@ def test_compile_with_framework_version_15(session):
         job_name="compile-model",
     )
 
-    assert "1.5" in model.image_uri
+    assert model.image_uri is not None
```

Comment:
any specific reason we are not checking "1.5" here in image uri

Reply:
Since the uri is fetched from Neo API now, I think it would be suffice to just check that 
model.image_uri is equal to the uri inside the response.

Reply:
I think we should still check for 1.5 to make sure the uri we are receiving is as expected

Reply:
I don't think that is possible. Checking `1.5` requires API access within the unit tests. I changed it to make sure the `model.image_uri` is the same as what is inside the response.